core/config/presenters.go

```diff
@@ -10,3 +10,7 @@ import (
 func FriendlyBigInt(n *big.Int) string {
 	return fmt.Sprintf("#%[1]v (0x%[1]x)", n)
 }
+
+func FriendlyInt64(n int64) string {
+	return fmt.Sprintf("#%[1]v (0x%[1]x)", n)
+}
```

Comment:
nit: can FriendlyX be generic? (just less duplicate helpers)

maybe something like:
```go
func FriendlyNumber[N constraints.Integer | *big.Int](n N) string {
	return fmt.Sprintf("#%[1]v (0x%[1]x)", n)
}
```

Reply:
great suggestion! thanks :)